contracts/lacchain.system/ricardian/lacchain.system.contracts.md

```diff
@@ -0,0 +1,399 @@
+
+<h1 class="contract">onerror</h1>
+
+---
+spec_version: "0.2.0"
+title: On Error Action
+summary: 'This action is delivered to the sender of a deferred transaction'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+On error action, notification of this action is delivered to the sender of a deferred transaction
+when an objective error occurs while executing the deferred transaction.
+
+<h1 class="contract">addentity</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Entity
+summary: 'Add to the network a Partner and Non Partner entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to Add to the network a Partner and Non Partner entity.
+
+<h1 class="contract">rmentity</h1>
+
+---
+spec_version: "0.2.0"
+title: Remove Entity
+summary: 'Remove from the network a Partner and Non Partner entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to remove from the network a Partner and Non Partner entity.
+
+<h1 class="contract">addvalidator</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Validator
+summary: 'Add to the network a Validator for the entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to Add to the network a Validator for the entity.
+
+<h1 class="contract">addwriter</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Writer
+summary: 'Add to the network a Writer node for a Partner entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to Add to the network a Writer node for a Partner entity.
+
+<h1 class="contract">addboot</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Boot Node
+summary: 'Add a Boot Node to the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to add a Boot Node to the network.
+
+<h1 class="contract">addobserver</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Observer Node
+summary: 'Add a Observer Node to the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to add a Observer Node to the network.
+
+<h1 class="contract">netaddgroup</h1>
+
+---
+spec_version: "0.2.0"
+title: Create a Group of Nodes
+summary: 'Create a Group of Nodes to the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to add a Group of Nodes to the network.
+
+<h1 class="contract">rmnode</h1>
+
+---
+spec_version: "0.2.0"
+title: Remove a Node
+summary: 'Remove a Node from the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to remove a Node from the network.
+
+<h1 class="contract">netrmgroup</h1>
+
+---
+spec_version: "0.2.0"
+title: Remove a Group of Nodes
+summary: 'Remove a Group of Nodes from the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to remove a Group of Nodes from the network.
+
+<h1 class="contract">netsetgroup</h1>
+
+---
+spec_version: "0.2.0"
+title: Link a Node
+summary: 'Link a Node with a list of Groups to the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to link a Node with a list of Groups to the network.
```

Comment:
I believe it is the permisioning committee that can call net set  group

Reply:
it is also linking groups with groups , not nodes with groups (please check)

Reply:
The action header
![image](https://user-images.githubusercontent.com/28828796/126524938-3fd55fe1-5544-4317-b7ec-d2ea8de211f6.png)